sdk/keyvault/Azure.Security.KeyVault.Keys/src/Cryptography/Client/CryptographyClient.cs

```diff
@@ -0,0 +1,213 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Azure.Security.KeyVault.Cryptography.Client
+{
+    using Azure.Core;
+    using Azure.Core.Pipeline;
+    using Azure.Security.KeyVault.Cryptography.Base;
+    using Azure.Security.KeyVault.Cryptography.CryptoProvider;
+    using Azure.Security.KeyVault.Cryptography.Interface;
+    using Azure.Security.KeyVault.Cryptography.Utilities;
+    using Azure.Security.KeyVault.Keys;
+    using System;
+
+    /// <summary>
+    /// The Cryptography Client 
+    /// </summary>
+    public class CryptographyClient
+    {
+        #region const
+        const string DEFAULT_KV_SCOPE_URI = @"https://vault.azure.net/.default";
+        #endregion
+
+        #region fields
+
+        #endregion
+
+        #region Properties
+
+        /// <summary>
+        /// 
+        /// </summary>
+        public TokenCredential Credentials { get; }
```

Comment:
We don't expose credentials from any other clients we should remove this property

Reply:
sure will do it, this was again added looking at v1 client.